src/views/PersonProfile/PersonProfile.js

```diff
@@ -207,7 +251,7 @@ export default function PersonProfile() {
                             </GridItem>
                             <GridItem xs={12} sm={12} md={2}>
                               <CustomInput
-                                labelText="Apartment number"
+                                labelText="Apt. number"

```

Comment:
Same with this

Reply:
I actually don't understand why that got added to the diff

Reply:
I did it because the label was line wrapping